src/main/java/com/spotinst/sdkjava/model/converters/ElastigroupConverterSuspendAzure.java

```diff
@@ -0,0 +1,30 @@
+package com.spotinst.sdkjava.model.converters;
```

Comment:
Why this separate class is needed?

Reply:
moved this to ElastigroupCoverterAzure class